### PR TITLE
added layout and join google group functionality #6418

### PR DESCRIPTION
**Pull request by gautampanday**

### CODE REVIEW

File: theme/static/css/style.css [hunk 1]
```diff
@@ -1,4 +1,5 @@
 /* Navbar css starts */
+/* Navbar css starts */
```
Comment (GauravButola) [inline theme/static/css/style.css]: Remove.

File: theme/static/css/style.css [hunk 2]
```diff
@@ -507,4 +522,77 @@ line-height: 70px;
...
+.icon-envelope {
+  font-size: 18px;
+}
+
+.tryton-rss {
+  padding:18px 40px 18px 0;
+  border-top:1px solid #CFE3DC;
+}
+
+.list-font {
```
Comment (GauravButola) [inline theme/static/css/style.css]: list-color or maybe color-black if all it does is make the color dark.

File: theme/templates/community.html [hunk 3]
```diff
@@ -1,63 +1,325 @@
...
+  <div class="col-sm-12">
+    <div class="tryton-rss">
+      <a data-toggle="collapse" href="#mailinglist">
+        <h4 class="margin-0 common-color heading">Few more mailing lists <small><span class="pull-right glyphicon glyphicon-plus common-color"></span></small></h4>
+      </a>
+      <div id="mailinglist" class="collapse">
+        <br>
+        <h5>Follow major announcements</h5>
+        <h3 class="margin-0">
+          <small><a class="mailing-list list-font" href="http://groups-announce.tryton.org/">tryton-announce</a></small>
```
Comment (GauravButola) [inline theme/templates/community.html]: Why using list-font if mailing-list will override list-font color anyway.

File: theme/templates/community.html [hunk 4]
```diff
@@ -1,63 +1,325 @@
...
+        <br>
+        <h5>Get notified of change sets
+        </h5>
+        <h3 class="margin-0">
+          <small><a class="mailing-list list-font" href="http://groups-commit.tryton.org/">tryton-commit</a></small>
+        </h3>
+        <br>
+        <h5>Contribute to the development of Tryton</h5>
+        <h3 class="margin-0">
+          <small><a class="mailing-list list-font" href="http://groups-contrib.tryton.org/">tryton-contrib</a></small>
```
Comment (GauravButola) [inline theme/templates/community.html]: Why using list-font if mailing-list will override list-font color anyway.

**Comment by GauravButola** [comment]
LGTM.

File: theme/templates/community.html [hunk 5]
```diff
@@ -1,63 +1,326 @@
...
-          <ul>
-            <li>English: <a href="irc://irc.freenode.net/tryton">#tryton</a> (<a href="http://www.tryton.org/~irclog/">{% trans %}Archives{% endtrans %}</a>)</li>
-            <li>Deutsch: <a href="irc://irc.freenode.net/tryton.de">#tryton.de</a> (<a href="http://www.tryton.org/~irclog/de/">{% trans %}Archives{% endtrans %}</a>)</li>
-            <li>Español: <a href="irc://irc.freenode.net/tryton-es">#tryton-es</a> (<a href="http://www.tryton.org/~irclog/es/">{% trans %}Archives{% endtrans %}</a>)</li>
-            <li>Français: <a href="irc://irc.freenode.net/tryton-fr">#tryton-fr</a> (<a href="http://www.tryton.org/~irclog/fr/">{% trans %}Archives{% endtrans %}</a>)</li>
-            <li>РУССКИЙ: <a href="irc://irc.freenode.net/tryton-ru">#tryton-ru</a> (<a href="http://www.tryton.org/~irclog/ru/">{% trans %}Archives{% endtrans %}</a>)</li>
+            <a class="btn btn-default white-text-color irc-detail" href="#" >{% trans %}View archive{% endtrans %}</a>
+          </div>
+          <div class="col-xs-4 col-sm-3 col-md-2 col-lg-2 text-center tryton-interface">
+            <a class="common-color mailing-list">Commits</a>
```
Comment (tarunbhardwaj) [inline theme/templates/community.html]: trans

File: theme/templates/community.html [hunk 6]
```diff
@@ -1,63 +1,326 @@
...
+      <br>
+      <br>
+      <div class="row">
+        <div class="col-sm-4 grid-spaces">
+          <div class="row">
+            <div class="col-sm-4">
+              <img src="{{ STATIC }}images/community/wiki.png" alt="community" class="img img-responsive content-center-xs">
+            </div>
+            <div class="col-sm-8 text-center-xs">
+              <h4 class="heading"><a class="common-color mailing-list" href="http://wiki.tryton.org/">Wiki</a></h4>
```
Comment (tarunbhardwaj) [inline theme/templates/community.html]: trans

File: theme/static/css/style.css [hunk 7]
```diff
@@ -537,4 +551,73 @@ line-height: 70px;
   border-color:#1ac6ff; 
 }
 
-/* download.html css end*/
\ No newline at end of file
+/* download.html css end*/
+
+/*community css starts */
+.input-resize {
```
Comment (tarunbhardwaj) [inline theme/static/css/style.css]: Not used

File: theme/static/css/style.css [hunk 8]
```diff
@@ -537,4 +551,73 @@ line-height: 70px;
...
+  padding: 23px 0 23px 7px;
+}
+
+.icon-envelope {
```
Comment (tarunbhardwaj) [inline theme/static/css/style.css]: Not used

File: theme/static/css/style.css [hunk 9]
```diff
@@ -537,4 +551,73 @@ line-height: 70px;
...
+
+.grid-spaces {
+  padding-bottom: 35px;
+}
+
+.social-desc {
+  padding-top: 38px;
+}
+
+.suscribe-button {
```
Comment (tarunbhardwaj) [inline theme/static/css/style.css]: typo and not used

File: theme/templates/community.html [hunk 10]
```diff
@@ -1,63 +1,314 @@
...
+    <div class="item active">
+      <div class="community-banner banner"></div>
+    </div>
+    <div class="item">
+      <div class="community-banner2 banner"></div>
+    </div>
+    <!-- Controls -->
+    <a class="left carousel-control" href="#community-banner" role="button" data-slide="prev">
+      <span class="glyphicon glyphicon-chevron-left" aria-hidden="true"></span>
+      <span class="sr-only">Previous</span>
```
Comment (tarunbhardwaj) [inline theme/templates/community.html]: trans

File: theme/templates/community.html [hunk 11]
```diff
@@ -1,63 +1,314 @@
...
+    </a>
+    <a class="right carousel-control" href="#community-banner" role="button" data-slide="next">
+      <span class="glyphicon glyphicon-chevron-right" aria-hidden="true"></span>
+      <span class="sr-only">Next</span>
```
Comment (tarunbhardwaj) [inline theme/templates/community.html]: trans

File: theme/templates/papers.html [hunk 12]
```diff
@@ -0,0 +1,187 @@
...
+    </div>
+  </div>
+</section>
+<section class="section-space">
+  <div class="container">
+    <h2 class="heading">Presentations &amp; Papers</h2>
+      <h3>Presentations: TUB2013</h3>
+      <ul>
+        <li>
+          <a href="//downloads.tryton.org/TUB2013/no_localization/">Why you should not do localization modules?</a>, <em>Cédric Krier</em>,
```
Comment (GauravButola) [inline theme/templates/papers.html]: Don't we need trans here? @tarunbhardwaj

**Comment by GauravButola** [comment]
Separate commit for papers page

File: theme/templates/community.html [hunk 13]
```diff
@@ -1,63 +1,314 @@
...
+    </div>
+    <div class="col-sm-8 col-sm-offset-2">
+     <br>
+     <br>
+     <div class="col-sm-12">
+      <div class="tryton-rss">
+        <a data-toggle="collapse" href="#mailinglist">
+          <h4 class="margin-0 common-color heading">{% trans %}Few more mailing lists{% endtrans %} <small><span class="pull-right glyphicon glyphicon-plus common-color"></span></small></h4>
+        </a>
+        <div id="mailinglist" class="collapse">
```
Comment (GauravButola) [inline theme/templates/community.html:72]: I think creating id would invoke error in html lint.

**Comment by GauravButola** [comment]
LGTM

**Pull request merged**